recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "ProDy" %}
-{% set version = "2.2.0" %}
+{% set version = "2.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/prody/ProDy/archive/v{{ version }}.tar.gz
-  sha256: 9d2185a42fe27328cf35c86db05c13df086ea8e273a4a85a747cc999b0f8f4a5
+  sha256: f42dc2a9715d96345d4fdbfee451b10c19b696246059d0973654fe1c76c557bf
```

Comment:
please update again

Reply:
should be ca760eb8a17d27fdcb06d17e7a09ce2a05e7371a055f69278599dfbd8fd9525f, I think. I just copied this from PyPI

Reply:
the same shows up using Get-fileHash dist/ProDy-2.4.0.tar.gz on Powershell just with capital letters

Reply:
@jamesmkrieger please feel free to update yourself, you can also try the command mentioned in the PR text to update the PR

Reply:
Ok, I’ve put the command in. I just realised the PyPI wouldn’t be the right one because GitHub makes its own tar.gz